<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowAll</class>
 <widget class="QMainWindow" name="MainWindowAll">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actAll_new"/>
   <addaction name="actAll_open"/>
   <addaction name="actAll_close"/>
   <addaction name="actAll_MDI"/>
   <addaction name="actAll_tile"/>
   <addaction name="actAll_initMode"/>
   <addaction name="actAll_compare"/>
   <addaction name="actAll_line"/>
  </widget>
  <action name="actAll_new">
   <property name="text">
    <string>新建文件</string>
   </property>
  </action>
  <action name="actAll_open">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actAll_close">
   <property name="text">
    <string>关闭全部</string>
   </property>
  </action>
  <action name="actAll_MDI">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>MDI</string>
   </property>
  </action>
  <action name="actAll_tile">
   <property name="text">
    <string>平铺</string>
   </property>
  </action>
  <action name="actAll_initMode">
   <property name="text">
    <string>级联</string>
   </property>
  </action>
  <action name="actAll_compare">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>整体对比</string>
   </property>
  </action>
  <action name="actAll_line">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>段对比</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actAll_close</sender>
   <signal>triggered()</signal>
   <receiver>mdiArea</receiver>
   <slot>closeAllSubWindows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
